resources/config.py

```diff
@@ -42,35 +45,48 @@ def get(self):
                 description: Static configuration global statistics
                 content:
                   application/json:
-                    schema: ConfigStatsSchema
+                    schema: ConfigStatsResponseSchema
         """
-        from_time = request.args.get('range', '*')
+        schema = ConfigStatsRequestSchema()
+        params = schema.load(request.args)
+
+        if params and params.errors:
```

Comment:
can `params` be Null? Maybe we should check `it nof params or params.error`?

Reply:
can't be null, good catch :+1: